src/patterns/fundamentals/layout/flag.hbs

```diff
@@ -0,0 +1,20 @@
+---
+name: Flag Object
```

Comment:
Should we make a PR on Scales to update Flag to use flex instead of table?

Reply:
I added an issue on scales for this.